trillian/ctfe/instance.go

```diff
@@ -62,28 +63,43 @@ type InstanceOptions struct {
 	STHStorage MirrorSTHStorage
 }
 
-// SetUpInstance sets up a log (or log mirror) instance using the provided
-// configuration, and returns a set of handlers for this log. This is the
-// standard way of setting up a log instance.
-func SetUpInstance(ctx context.Context, opts InstanceOptions) (*PathHandlers, error) {
-	handlers, _, err := SetUpInstanceAndGetter(ctx, opts)
-	return handlers, err
+// Instance is a set up log/mirror instance. It must be created with the
+// SetUpInstance call.
+type Instance struct {
+	Opts      InstanceOptions
```

Comment:
This one can probably be removed for now, as it's a copy of `li.instanceOpts`, and so far not used externally.

Reply:
OK I won't comment on the doc string then. TODO for removal in a followup?